Eleventy v3 (#902)
thegetty/quire: packages/11ty/CHANGELOG.md

- `Fixed`: for any bug fixes
- `Removed`: for deprecated features removed in this release

> ## []

Author: ```suggestion
## [unreleased]
```

### Bumped

- Update dependency pins for `@11ty/eleventy`, `@11ty/eleventy-plugin-vite` to support Eleventy v3

> This release is a somewhat major change to eleventy's version 3, which requires javascript files to use ESM imports and associated abstractions (eg, import.meta.url). For most quire users this should not affect much but some integrators that make customizations to quire code or import packages not in a quire publication's dependency tree may need to address ESM conformance in their custom modules.

Author: ```suggestion

### Breaking

This release is updates the *major* version of `eleventy`, which requires using ECMAScript module (ESM) syntax for JavaScript code. Specifically, ESM `import` statements to use other modules and use the `import.meta.url` property for a module's file path from within that module. For most Quire users this update should be transparent. For publications that have customized the quire code or import packages that are not already in the publication's package dependency tree changes may be needed to address ESM conformance to the custom modules.
```

- `Fixed`: for any bug fixes
- `Removed`: for deprecated features removed in this release

## [unreleased]

### Bumped

- Update dependency pins for `@11ty/eleventy`, `@11ty/eleventy-plugin-vite` to support Eleventy v3

> 
> 
> ### Breaking
> 
> This release is updates the *major* version of `eleventy`, which requires using ECMAScript module (ESM) syntax for JavaScript code. Specifically, ESM `import` statements to use other modules and use the `import.meta.url` property for a module's file path from within that module. For most Quire users this update should be transparent. For publications that have customized the quire code or import packages that are not already in the publication's package dependency tree changes may be needed to address ESM conformance to the custom modules.
> 

Author: ```suggestion

### Breaking

This release is updates the *major* version of `eleventy`, which requires using ECMAScript module (ESM) syntax for JavaScript code. Specifically, ESM `import` statements to use other modules and the `import.meta.url` property for a module's file path from within that module. For most Quire users this update should be transparent. For publications that have customized the quire code or import packages that are not already in the publication's package dependency tree changes may be needed to address ESM conformance to the custom modules.
```
